include/triton/Tools/LinearLayout.h

```diff
@@ -575,9 +575,20 @@ class LinearLayout {
     return *this;
   }
 
-  // divideLeft and divideRight are the inverses of operator*.
+  // divideLeft and divideRight are the inverses of operator *.
```

Comment:
In C++ it's called `operator*`, not `operator *`

Reply:
Ah, it's a GPT bug...

Reply:
I didn't meant to change this line